content/_data/changelogs/weekly.yml

```diff
@@ -6948,6 +6948,97 @@
       authors:
         - jvz
 
+- version: 2.229
+  date: 2020-03-29
+  changes:
+  - type: major bug
+    category: major bug
+    pull: 4602
+    issue: 61688
+    authors:
+    - daniel-beck
+    message: |-
+      Actually load a previously saved global build discarder configuration after restart.
+  - type: major bug
+    category: regression
+    pull: 4607
+    issue: 61692
+    authors:
+    - daniel-beck
+    message: |-
+      Fix proxy form validation when a password is set (regression in 2.205).
+  - type: rfe
+    category: rfe
+    pull: 4554
+    authors:
+    - NextTurn
+    message: |-
+      Update .NET version checks to be more correct for modern .NET versions.
+  - type: rfe
+    category: rfe
+    pull: 4569
+    issue: 61455
+    authors:
+    - amuniz
+    - timja
+    message: |-
+      <code>About Jenkins</code> management link is now accessible to users with <code>Overall/Manage</code> or <code>Overall/SystemRead</code> (as well as the usual <code>Overal/Administer</code>).
+  - type: bug
+    category: bug
+    pull: 4608
+    authors:
+    - daniel-beck
+    message: |-
+      Robustness: Don't throw a <code>NullPointerException</code> when trying to convert <code>null</code> to <code>Secret</code>.
+  - type: bug
+    category: bug
+    pull: 4601
+    issue: 61409
```

Comment:
YAGNI, it is referenced below

Reply:
Is your concern that I don't need to list the issue at the same level as the type, category, and pull request because it is also mentioned in the references?  I've used that redundancy in the past on the assumption that we wanted the entry at the type/category level for completeness and we wanted it in the references for visibility.

If that's not the concern, can you explain further?

Thanks!